home/whatsapp.py

```diff
@@ -183,97 +279,86 @@ def create_whatsapp_template_submission(
     return components
 
 
-def create_whatsapp_template_image(image_obj: Image) -> dict[str, Any]:
-    image_handle = upload_image(image_obj)
+###### OLD CONTENTPAGE EMBEDDED TEMPLATE CODE ABOVE ######
+###### NEW STANDALONE TEMPLATE CODE BELOW ######
+
 
+def create_standalone_template_body_components(
+    message: str,
+    quick_replies: Iterable[str] = (),
+    example_values: Iterable[str] | None = None,
+) -> list[dict[str, Any]]:
+    """
+    Create the body and buttons components of a WhatsApp template submission
+    request, but not the images because those need to be uploaded separately.
+    """
+
+    # body: dict[str, Any] = {"type": "BODY", "text": body_text}
+
+    components: list[dict[str, Any]] = []
+
+    if example_values:
+        components.append(
+            {
+                "type": "BODY",
+                "text": message,
+                "example": {
+                    "body_text": [example_values],
+                },
+            }
+        )
+    else:
+        components.append({"type": "BODY", "text": message})
+
+    if quick_replies:
+
+        buttons = []
+        for button in quick_replies:
+            buttons.append({"type": "QUICK_REPLY", "text": button})
+        components.append({"type": "BUTTONS", "buttons": buttons})
+
+    return components
+
+
+def create_standalone_template_header_components(
```

Comment:
I think it's a little confusing that `create_standalone_template_body_components` returns a list of components, but this returns just a single component. Typing might help, but I feel it's too easy to make the mistake of calling `append` for both of these, and ending up with an invalid list of components.

Maybe we could just remove the `s` at the end of this function name, but that doesn't seem noticeable enough. 

So maybe the best option is for this function to also return a list, so that it's consistent, just that this function will only return a single item in that list.